core/templates/pages/about-page/about-page.component.ts

```diff
@@ -16,75 +16,94 @@
  * @fileoverview Controllers for the about page.
  */
 
-require(
-  'components/common-layout-directives/common-elements/' +
-  'background-banner.component.ts');
+import { Component, OnInit } from '@angular/core';
+import { downgradeComponent } from '@angular/upgrade/static';
 
-require('domain/utilities/url-interpolation.service.ts');
+import { I18nLanguageCodeService } from 'services/i18n-language-code.service';
+import { TranslateService } from 'services/translate.service';
+import { UrlInterpolationService } from
+  'domain/utilities/url-interpolation.service.ts';
+import { WindowRef } from
+  'services/contextual/window-ref.service.ts';
 
-angular.module('oppia').component('aboutPage', {
-  template: require('./about-page.component.html'),
-  controller: [
-    'UrlInterpolationService', 'WindowRef',
-    function(UrlInterpolationService, WindowRef) {
-      const ctrl = this;
-      const listOfNamesToThank = [
-        'Alex Kauffmann', 'Allison Barros',
-        'Amy Latten', 'Brett Barros',
-        'Crystal Kwok', 'Daniel Hernandez',
-        'Divya Siddarth', 'Ilwon Yoon',
-        'Jennifer Chen', 'John Cox',
-        'John Orr', 'Katie Berlent',
-        'Michael Wawszczak', 'Mike Gainer',
-        'Neil Fraser', 'Noah Falstein',
-        'Nupur Jain', 'Peter Norvig',
-        'Philip Guo', 'Piotr Mitros',
-        'Rachel Chen', 'Rahim Nathwani',
-        'Robyn Choo', 'Tricia Ngoon',
-        'Vikrant Nanda', 'Vinamrata Singal',
-        'Yarin Feigenbaum'];
-      // Define constant for each tab on the page.
-      ctrl.TAB_ID_ABOUT = 'about';
-      ctrl.TAB_ID_FOUNDATION = 'foundation';
-      ctrl.TAB_ID_CREDITS = 'credits';
+@Component({
+  selector: 'about-page',
+  templateUrl: './about-page.component.html'
+})
+export class AboutPageComponent implements OnInit {
+  listOfNames: string;
+  aboutPageMascotImgUrl: string;
+  activeTabName: string;
+  listOfNamesToThank = [
+    'Alex Kauffmann', 'Allison Barros',
+    'Amy Latten', 'Brett Barros',
+    'Crystal Kwok', 'Daniel Hernandez',
+    'Divya Siddarth', 'Ilwon Yoon',
+    'Jennifer Chen', 'John Cox',
+    'John Orr', 'Katie Berlent',
+    'Michael Wawszczak', 'Mike Gainer',
+    'Neil Fraser', 'Noah Falstein',
+    'Nupur Jain', 'Peter Norvig',
+    'Philip Guo', 'Piotr Mitros',
+    'Rachel Chen', 'Rahim Nathwani',
+    'Robyn Choo', 'Tricia Ngoon',
+    'Vikrant Nanda', 'Vinamrata Singal',
+    'Yarin Feigenbaum'];
+  // Define constant for each tab on the page.
+  TAB_ID_ABOUT: string = 'about';
+  TAB_ID_FOUNDATION: string = 'foundation';
+  TAB_ID_CREDITS: string = 'credits';
+  ALLOWED_TABS: Array<string>;
```

Comment:
Why is it no longe constructed here?

Reply:
Missed that, shifted the initialization back to this line.